### PR TITLE
Upgrade explorer GraphQL from 3.0.0 to 4.7.1

### PR DESCRIPTION
**Pull request by area363**
## Changes
* Bump GQL to 4.7.1
* Fix test to fit the new version

### CODE REVIEW

File: Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs [hunk 1]
```diff
@@ -53,28 +54,32 @@ public async void Query()
...
-            Assert.Equal(
-                ByteUtil.Hex(block.Nonce.ToByteArray()),
-                resultData["nonce"]);
-            Assert.Equal(
-                new DateTimeOffsetGraphType().Serialize(block.Timestamp),
-                resultData["timestamp"]);
-            Assert.Equal(
-                ByteUtil.Hex(block.StateRootHash.ToByteArray()),
-                resultData["stateRootHash"]);
+            if (resultData != null)
```
Comment (dahlia) [inline Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs:60]: In what cases `resultData` is not present?

File: Libplanet.Explorer.UnitTests/GraphTypes/ByteStringTypeTest.cs [hunk 2]
```diff
@@ -30,7 +30,9 @@ public void ParseValue(object value, object parsed)
         [InlineData("beef", new byte[] { 0xbe, 0xef })]
         public void ParseLiteral(string stringValue, object parsed)
         {
-            Assert.Equal(parsed, _type.ParseLiteral(new StringValue(stringValue)));
+            var actual =
+                stringValue != null ? _type.ParseLiteral(new StringValue(stringValue)) : null;
```
Comment (dahlia) [inline Libplanet.Explorer.UnitTests/GraphTypes/ByteStringTypeTest.cs:34]:
```suggestion
                stringValue is { } v ? _type.ParseLiteral(new StringValue(v)) : null;
```

File: Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs (hunk 1 above)
Comment (area363) [reply Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs:60]: I checked again just now and it appears that there isn't a case where the `resultData` is null. I might have included this code before fixing another API. I'll remove this part along with https://github.com/planetarium/libplanet/pull/1690/files#diff-aeb0e1c689f74d1eefa3a1ca8914edc2c4c379114561a8011f9b52e610d5b5a1R45-R46. Thanks for checking. 🙏

Comment (dahlia) [reply Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs:60]: Note that there is another similar pattern in `TransactionTypeTest.Query()` method as well.

**Review by dahlia — APPROVED** [review]

**Review by dahlia — CHANGES_REQUESTED** [review]
Wait, the build apparently failed.  I'm going to take a look.

**Review by dahlia — APPROVED** [review]
It was due to no luck.  Rerun fixed the build.

**Pull request merged**